scripts/1.download.py

```diff
@@ -0,0 +1,52 @@
+
+# coding: utf-8
```

Comment:
not sure what you mean? I am saying that running either example (either the`TP53` or `Hippo`) directly after cloning would output the result without having to first download the data. With this change (and the corresponding change in [lines 69-80 of `algorithms/SGDClassifier-Master.py](https://github.com/cognoma/machine-learning/pull/42/files#diff-085839bf1754a46003c4dfbba1c979f8L69) the user would need to run `scripts/1.download.py` first. 

Hmmm...I guess this would be ok and moving toward a more succinct workflow would be good. A user would be able to see the ipython notebook anyway with the results embedded. Ok I think I talked myself out - no longer concerned!